spec/spec_helper.rb

```diff
@@ -4,6 +4,7 @@
 require 'bolt/logger'
 require 'logging'
 require 'rspec/logging_helper'
+require_relative '../vendored/require_vendored'
```

Comment:
Why is this needed? Doesn't load_puppet for particular tests cover it?

Reply:
Or... why did this work before?

Reply:
we may try to reset puppet's settings before any test has called load_puppet.

Reply:
I'm not sure how it worked before